wellcomecollection/editorial-photography-ingest
# Download from s3

## What does this change?

This adds the facility to download all the relevant objects from a folder associated with a shoot (assuming that the folder is available, i.e. has already been restored from Glacier).

## How to test

This is covered by unit tests using Moto and pyfakefs.  You can see it for real life by running  `python download.py PBBD_TEST` when authenticated as a user with permission to download from the `platform` account.

## Have we considered potential risks?

This isn't running anywhere just yet, so no problems until we actually do use it and it turns out not to do what we need.
